product-service/serverless.ts

```diff
@@ -0,0 +1,184 @@
+import { newProductSchema } from "./src/handlers/createProduct/handler";
+
+const serverlessConfigurationProductService = {
+  service: 'product-service',
+  provider: {
+    name: 'aws',
+    runtime: 'nodejs20.x',
+    stage: 'dev',
+    region: 'eu-west-1',
+    environment: {
+      PRODUCTS_TABLE: 'Products',
+      STOCK_TABLE: 'Stock',
+      REGION: 'eu-west-1',
+      SNS_ARN: 'arn:aws:sns:eu-west-1:637423586618:createProductTopic'
```

Comment:
Not a good practice to commit your account (I can't say that it's quite vulnerable information, but it can be changed from environment to environment)

Usually it's do the following way:
You setup AWS profile in your computer with specific name: https://medium.com/nerd-for-tech/configuration-and-credential-file-settings-in-aws-cli-61c7ff0a1cd6

You can set your profile name in serverless.ts
<img width="778" alt="image" src="https://github.com/dilbar-akkaya/backend_shop/assets/90376926/641e7a24-eaaa-4271-a31b-22afee48f187">

Serverless just reference the account that is currently using
<img width="975" alt="image" src="https://github.com/dilbar-akkaya/backend_shop/assets/90376926/05bc0487-a750-4824-a5e6-3128dcb993ba">

Now every time you will run serveless command it will use your setup profile

Reply:
done